### DISCUSSION TITLE
Trivy 0.56.1 ignores #tfsec:ignore:aws-vpc-no-public-egress-sgr (Trivy 0.55.2 respects it)

### DISCUSSION THREAD

**Discussion by mig5** [aquasecurity/trivy#7682] [Q&A]
### Question

I have statements in TF code like this:

```
[..]
cidr_blocks = ["0.0.0.0/0"] #tfsec:ignore:aws-vpc-no-public-egress-sgr
[..]
```

In Trivy 0.55.2, these findings are ignored as expected by the annotation.

I upgraded to Trivy 0.56.1 and I get CRITICAL findings instead. It is ignoring the `#tfsec:ignore:aws-vpc-no-public-egress-sgr`.

If I downgrade back to Trivy 0.55.2 and delete my `~/.cache/trivy`, everything works again.

Anyone else experiencing this?

I'm running Ubuntu 20.04 amd64 and running Trivy from the filesystem (not via Docker), Trivy is installed via apt.

### Target

Filesystem

### Scanner

Misconfiguration

### Output Format

None

### Mode

Standalone

### Operating System

Ubuntu 20.04

**Comment by nikpivkin** [comment]
Hi @mig5 ! 

We recently migrated our checks from Go to Rego and forgot to account for aliases. This will be fixed soon.

**Comment by nikpivkin** [comment]
Track https://github.com/aquasecurity/trivy/issues/7683
